src/dart_array.rs

```diff
@@ -14,6 +14,22 @@ pub struct DartArray {
     inner: Box<[*mut DartCObject]>,
 }
 
+impl<T: IntoDart, const N: usize> From<[T; N]> for DartArray {
+    fn from(vec: [T; N]) -> Self {
+        let vec: Vec<_> = IntoIterator::into_iter(vec)
+            // convert them to dart objects
+            .map(IntoDart::into_dart)
+            // box them to be transferred to dart
+            .map(Box::new)
+            // as pointers
+            .map(Box::into_raw)
+            // then collect everything
+            .collect();
+        let inner = vec.into_boxed_slice();
+        Self { inner }
+    }
+}
```

Comment:
For the zero-copy to be implemented for this, we shall make use of the `ZeroCopyBuffer`, however, since arrays are **mostly** stored in the stack, it is very likely to be like that for now ( a copy is required).